experiments/pages/lightningModalLauncher/showScreenFlowModalController.js

```diff
@@ -0,0 +1,22 @@
+({
+	handleShowModalFlowEvent: function(component, event, helper) {
+        let name = event.getParam("name");
+        if (name != 'showModalFlow') return;
+        let data = event.getParam("data");
+
+        component.set("v.showModal", true);
+        let flow = component.find("flow");
+        flow.startFlow(data.flowName, data && data.params ? data.params : null);
+    },
+    handleFlowStatusChange : function(component, event, helper) {
+        if (event.getParam('status') === 'FINISHED') {
+            component.set("v.showModal", false);
+        }
+//        if (event.getParam('currentStage') === 'Case_Created') {
```

Comment:
Typically we prefer to remove commented out code unless there is a specific reason for leaving it in (e.g. it's part of the explanation of why we did the way that isn't commented out), and we always leave an explanation of why it shouldn't be removed in a comment about it. I don't think this is used anywhere, so unless it's part of an explanation, I say delete it.